src/activity.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2017-2019, The rav1e contributors. All rights reserved
+//
+// This source code is subject to the terms of the BSD 2 Clause License and
+// the Alliance for Open Media Patent License 1.0. If the BSD 2 Clause License
+// was not distributed with this source code in the LICENSE file, you can
+// obtain it at www.aomedia.org/license/software. If the Alliance for Open
+// Media Patent License 1.0 was not distributed with this source code in the
+// PATENTS file, you can obtain it at www.aomedia.org/license/patent.
+
+use crate::frame::*;
+use crate::tiling::*;
+use crate::util::*;
+
+#[derive(Debug, Default, Clone)]
+pub struct ActivityMask {
+  variances: Vec<Vec<f64>>,
+  // Width and height of the original frame that is masked
+  width: usize,
+  height: usize,
+  // Side of unit (square) activity block in log2
```

Comment:
Oh, there it was already, good work. I missed this with seeing just impl { } below.

Reply:
Thanks :smile: